StickyTabBarViewController/Classes/MainTabBarController.swift

```diff
@@ -51,6 +55,12 @@ open class StickyViewControllerSupportingTabBarController: UITabBarController, S
         collapsableVCFlow!.heightConstraint = heightConstraint
 
         collapsableVCFlow!.didMove(toParent: self)
+        
+        if isFullScreenOnFirstAppearance {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) {
```

Comment:
Unfortunately not having this 0.01 seconds of delay results in a weird behaviour, where child VC doesn't animate to expanded state but rather jumps as full screen. I will try to come up with a better solution. @Thurman1776-

Reply:
🤔 I quickly removed it to see how the issue looks but I was not able to reproduce it. It's fine to keep it for now tho 👍

Reply:
It actually happens only on the first time, so if you remove asyncAfter block, then run the app. You should see the jumping behaviour on the first configuration. Then if you remove and add again, it doesnt happen 😅